File: Back_end/src/main/java/com/example/CRUD/entity/Task.java
package com.example.CRUD.entity;

import jakarta.persistence.*;

@Entity
@Table(name = "Tasks_table")
public class Task {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    @Column
    private String task;


    public Task(String task) {
        this.task=task;
    }

    public Task() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setTask(String task)
    {
        this.task=task;
    }
    public String getTask()
    {
        return task;
    }
}

File: Front_end/src/app/Task.ts
export class Task{
    id=0
    task=''

     constructor (task:string)
    {
    
        this.task=task;
    }
setid(id:number)
{
  this.id=id;
}
getid()
{
    return this.id
}
settask(task:string)
{
 this.task=task
}
gettask()
{
    return this.task;
}
}